lib/spack/spack/util/executable.py

```diff
@@ -8,12 +8,17 @@
 import shlex
 import subprocess
 import sys
+from sys import platform as _platform
 
 from six import string_types, text_type
 
 import llnl.util.tty as tty
 
 import spack.error
+from spack.util.path import Path, normalize_path, system_path_filter
+
+is_windows = _platform == 'win32'
```

Comment:
Just use sys.platform here and remove from sys... above. Actually, I don't think `is_windows `is used in this file

Reply:
I think you're right, will amend appropriately.